syscall/syscall.c

```diff
@@ -537,6 +548,14 @@ OE_WEAK OE_WEAK OE_DEFINE_SYSCALL3(SYS_lseek)
     return oe_lseek(fd, off, whence);
 }
 
+OE_WEAK OE_DEFINE_SYSCALL2(SYS_lstat)
+{
```

Comment:
Can you check why lstat is needed? Instead of just fulfilling the link requirement, we should check how it's used, and implement the ocall (with test) if necessary.

Reply:
lstat() is called in fstatat.c and fstatat.c is needed by stat.c. A runtime call to lstat is not expected..